resource_booking/models/resource_booking.py

```diff
@@ -431,7 +431,9 @@ def _prepare_meeting_vals(self):
             location=self.location,
             videocall_location=self.videocall_location,
             name=self.name
-            or self._get_name_formatted(self.partner_ids[0], self.type_id),
+            or self._get_name_formatted(self.partner_ids[0], self.type_id)
+            if self.partner_ids
+            else self.partner_id,
```

Comment:
Shouldn't this go inside the function?
```suggestion
            or self._get_name_formatted(
                self.partner_ids[0] if self.partner_ids else self.partner_id,
                self.type_id
            )
```

Reply:
I see your point. I’ll review it shortly.

Reply:
I finally went with the suggestion from @pedrobaeza using `partner_ids[:1]`, looks cleanner.